docs/content/dependencies/metadata-storage.md

```diff
@@ -12,17 +12,22 @@ a number of tables used for various purposes described below.
 The following metadata storage engines are supported:
 
 * Derby (default - only works if you have all processes running on the same node)
-* MySQL (io.druid.extensions:mysql-metadata-storage)
-* PostgreSQL (io.druid.extensions:postgresql-metadata-storage)
-
-To choose a metadata storage,
-
-1. Make sure Druid can pick up the extension files from either classpath or
-extensions directory, see [Including Extensions](../operations/including-extensions.html) for more information.
-
-2. set the `druid.extensions` configuration to include the extension for the
-metadata storage you plan to use. See below.
-
+* MySQL
+* PostgreSQL
+
+To choose the metadata storage type, set `druid.metadata.storage.type` to `derby`,
+`mysql` or `postgres`. Set other `druid.metadata.storage` configuration
+keywords as shown below to give Druid information about how to connect to
+the database.
+
+As discussed in [Including Extensions](../operations/including-extensions.html),
```

Comment:
The previous instructions seemed to be saying you'd need to set the druid.extensions configuration keyword even if you were picking up the extension files from the classpath.